three-node-cluster/start-performance.sh

```diff
@@ -124,39 +116,39 @@ while getopts "q:k:c:j:n:u:p:d:e:i:b:w:r:y:g:t:m:v:h" opts; do
     w)
         minimum_stack_creation_wait_time=${OPTARG}
         ;;
-    r)
-        concurrency=${OPTARG}
-        ;;
-    y)
-        jwt_token_client_secret=${OPTARG}
-        ;;
-    g)
-        jwt_token_user_password=${OPTARG}
```

Comment:
are we removing these configuration support ?

Reply:
No.
Since these configurations are needed only for the perf-test-is.sh file, instead of catching those values here, bypass it to the needed file.